### ISSUE TITLE
Tokenized card added automatically

### ISSUE THREAD

**Issue by RexNexstream** [braintree/braintree_android#217]
Hi 

In my project , I need to get the card infomation via CardIO and then tokenize the card and send payment method nonce to serve.

However , the behaviour for iOS and Android is different . 

In iOS , card will not added automatically to customer after card tokenization

`let braintreeClient = BTAPIClient(authorization: clientToken)!`
`let cardClient = BTCardClient(apiClient: braintreeClient)`
`let card = BTCard(number: cardInfo.cardNumber, expirationMonth: "\(cardInfo.expiryMonth)", expirationYear: "\(cardInfo.expiryYear)", cvv: nil)`
`cardClient.tokenizeCard(card) { (tokenizedCard, error) in
            // Communicate the tokenizedCard.nonce to your server, or handle error
        }`

But In Android, it added automatically

` mBraintreeFragment = BraintreeFragment.newInstance(this, clientToken);`
`CardBuilder cardBuilder = new CardBuilder()
                            .cardNumber(cardResult.cardNumber)
                            .expirationDate(cardResult.expiryMonth + "/" + cardResult.expiryYear);
                    Card.tokenize(mBraintreeFragment, cardBuilder);`

Please advise on this. How to prevent the tokenized card added automatically?
Both platform are using same client token.

Thank You

**Comment by quinnjn** [comment]
Hi @RexNexstream 

I'm not to sure I understand your question. Is your question related around vaulting the customers payment method i.e. saving the payment method to a customer?

Your best bet for this would be server side during `transaction.sale`. See our docs on [storing in vault](https://developers.braintreepayments.com/reference/request/transaction/sale/python#storing-in-vault).

If I misunderstood your question, can you rephrase it for me? Thanks!

**Comment by RexNexstream** [comment]
Ya. I only want to store the payment method in my Vault after transaction.sale.

But , android sdk auto store the payment method after card tokenization which the nonce have yet send to server side.

Need your advise on this issue . Thank you

**Comment by RexNexstream** [comment]
Ya .  This is what i need. Thank you .
